t/plugin/limit-count.t

```diff
@@ -834,3 +834,45 @@ passed
 [200, 200, 200, 503, 200, 200, 200, 503]
 --- no_error_log
 [error]
+
+
+
+=== TEST 24: set route(key: consumer_name)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "methods": ["GET"],
+                        "plugins": {
+                            "limit-count": {
+                                "count": 2,
+                                "time_window": 60,
+                                "rejected_code": 503,
+                                "key": "consumer_name"
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1980": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/hello"
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
```

Comment:
It would be better to add more test cases :)

Reply:
could you please give some test cases? so Jaycean would implement them

Reply:
@liuxiran @juzhiyuan done.